addons/web/static/tests/core/domain_selector_tests.js

```diff
@@ -375,4 +376,28 @@ QUnit.module("Components", (hooks) => {
         assert.containsOnce(target, ".o_domain_selector");
         assert.containsN(target, ".o_domain_leaf_edition > div", 2, "value should be hidden");
     });
+
+    QUnit.test("cache fields_get", async (assert) => {
+        assert.expect(2);
+
+        const mockRPC = (route, { method }) => {
+            if (method === "fields_get") {
+                assert.step("fields_get");
+            }
+        };
+
+        await makeMockServer(serverData, mockRPC);
+        const env = await makeTestEnv();
+
+        await mountComponent(DomainSelector, {
```

Comment:
Maybe have a mockRPC param instead of env ? This way you wouldn't need to build the env here (and it's closer to the makeView api)